README.md

```diff
@@ -3,6 +3,8 @@
 A **clickable** adaptation the Python Visualization Landscape slide from Jake
 VanderPlas' keynote at Pycon 2017. Made with Omnigraffle.
 
+Visible at http://http://rougier.github.io/python-visualization-landscape/landscape-colors.html
```

Comment:
Yep.
A small trick could help to have a 1:1 scale instead of a zoomed view of the page (I saw it before, but that's the kind of HTML/CSS tricks I can never remember, sorry)